src/extension.ts

```diff
@@ -271,6 +281,47 @@ function startWebhint(context: vscode.ExtensionContext): void {
             // Notify the server if a webhint-related configuration changes.
             fileEvents: vscode.workspace.createFileSystemWatcher('**/.hintrc'),
         },
+        middleware: {
+            executeCommand: async (command, args, next) => {
+                    const problemName = args[0] as string | undefined;
+                    const hintName = args[1] as string | undefined;
+
+                    switch (command) {
+                        case 'vscode-webhint/ignore-hint-project': {
+                            void ignoreHintPerProject(hintName);
+                            break;
+                        }
+                        case 'vscode-webhint/ignore-hint-global': {
+                            await ignoreHintGlobally(hintName, globalStoragePath || '');
+                            client.sendNotification(reloadAllProjectsConfigNotification);
```

Comment:
Since this effectively restarts the language server anyway, could we do the following to avoid having an extra contract between the host and language server (and wrap it as a `restartWebhint` helper)?
```ts
await stopWebhint();
startWebhint(context);
```